bin/varnishstat/varnishstat_options.h

```diff
@@ -36,14 +36,6 @@
 	    "Instead of presenting a continuously updated display,"	\
 	    " print the statistics to stdout."				\
 	)
-#define STAT_OPT_f							\
```

Comment:
Does this have to be redefined `VSC_OPT_f`?

Reply:
It is, but in the correct file.

Reply:
Our number one backport rule is don't break anything unless there is a security or safety issue. So breaking this parameter for a new feature is a no go. Your first PR did not have this change, so can we go back to that?

Reply:
I don't understand, `varnishstat_options.h` is a private header so this change increases the visibility of an option that should have been exposed from day one. Removing a `STAT_OPT_?` definition is not breaking change.

#3395 very much had this change too.

Reply:
I get that, but we also have a patch which does not do this header alteration (#3395 does not alter any headers). So this is definitely pushing the breakage boundary and the new feature boundary, simultaneously. If anything, leave the old `STAT_OPT_f` definition in place and simply add a new definition for `VSC_OPT_f`. I could even ask for 2 separate PRs, 1 for the fix and 1 for the new feature, but in the spirit of getting this thru, I will let this slide :)

Reply:
My bad, this change is not part of #3395, I could have sworn it was in both #3395 and #3396 since both PRs came with a bit of polish in the area.

HOWEVER that still doesn't change the fact that `varnishstat_options.h` is a PRIVATE header that has NO END USER except `varnishstat` because we DON'T INSTALL IT so this is NOT A BREAKING CHANGE. There is nothing to let slide, except how picky you are being on a non-issue.

I hope the ALL-CAPS will help you CONNECT THE DOTS.

Reply:
ok, this is an unfortunate response. Anyway, going to put this on hold for a bit and discuss with the backport team. We already finalized 6.0.7, so if this moves forward, we have plenty of time to get it on the next release.